src/tests/helper.ts

```diff
@@ -0,0 +1,122 @@
+import * as path from "path";
+import * as pfs from "@ts-common/fs";
+import { git, cli, devOps, avocado } from "@azure/avocado";
+
+export const create = async (name: string) => {
+  const tmpRoot = path.resolve(path.join("..", "avocado-tmp"));
+  if (!(await pfs.exists(tmpRoot))) {
+    await pfs.mkdir(tmpRoot);
+  }
+  const tmp = path.join(tmpRoot, name);
+
+  if (await pfs.exists(tmp)) {
+    await pfs.recursiveRmdir(tmp);
+  }
+
+  await pfs.mkdir(tmp);
+  return tmp;
+};
+
+export const createDevOpsEnv = async (name: string): Promise<cli.Config> => {
+  const tmp = await create(name);
+
+  // Create '"${tmp}/remote"' folder.
+  const remote = path.join(tmp, "remote");
+  await pfs.mkdir(remote);
+
+  const gitRemote = git.repository(remote);
+
+  // create a Git repository
+  await gitRemote({ init: [] });
+  await gitRemote({ config: ["user.email", "test@example.com"] });
+  await gitRemote({ config: ["user.name", "test"] });
+
+  // commit invalid 'specification/readme.md' to 'master'.
+  const specification = path.join(remote, "specification");
+  await pfs.mkdir(specification);
+  await pfs.writeFile(path.join(specification, "readme.md"), "");
+  await pfs.writeFile(
+    path.join(specification, "file1.json"),
+    `
```

Comment:
Can we move all the inline JSON as separate file?

Reply:
Do you mean after running the test we need to make a cleanup.

Reply:
This is just for mocking git repo . Only used once. So it's not needed to move inline json to file. 

Added cleanUp function.

Reply:
Sounds good.